modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -1084,6 +1084,60 @@ definition {
 		}
 	}
 
+	@description = "LPS-164948 - Verify that the Label, Output and Formula Builder can be edited after the System Object be saved"
+	@priority = 4
+	test CanEditFormulaFieldOnSystemObject {
+		property custom.properties = "feature.flag.LPS-164948=true";
+
+		task ("Given: A System Object with a formula field") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectSystemObject(label = "Commerce Order");
+
+			ObjectAdmin.goToFieldsTab();
+
+			ObjectAdmin.addObjectFieldViaUI(
+				fieldLabel = "Custom Formula Field",
+				fieldType = "Formula",
+				formulaFieldOption = "Decimal");
```

Comment:
Can we use the API instead of the UI here? If it's not possible, how much effort would it take to add this feature?

Reply:
This is currently only possible through the UI. I will ask for help to add this feature.